web/skins/classic/views/events.php

```diff
@@ -274,25 +294,25 @@
 }
 ?>
         <div id="contentButtons">
-          <button type="button" name="viewBtn" value="View" data-on-click-this="viewEvents" disabled="disabled">
+          <button type="button" name="viewBtn" value="View" onclick="viewEvents(this, 'markEids');" disabled="disabled">
```

Comment:
We are removing inline event listeners and banning them via CSP with script-src. I guess you accidentally reverted these.

Reply:
uh oh

Reply:
is your change merged to master? maybe that's why I'm not seeing those changes

Reply:
Never mind, stupid mistake. Will fix.